FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +20,51 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] clientResult in
+			guard self != nil else { return }
+			switch clientResult {
+			case let .success((data, response)):
+				let feedLoaderResult = RemoteFeedLoader.getFeedLoaderResult(data, response)
+				completion(feedLoaderResult)
```

Comment:
Suggestion:
Since `feedLoaderResult` does not add any contextual value not already present in `getFeedLoaderResult`, we can one-line this completion:
```suggestion
				completion(RemoteFeedLoader.getFeedLoaderResult(data, response))
```